### ISSUE TITLE
Write an "implementation white paper" with more specific details

### ISSUE THREAD

**Issue by michwill** [nucypher/nucypher-kms#58]

**Comment by jMyles** [comment]
So, what do we want for this white paper?

I'm imagining a paper that deals less with how the cryptography works and more about how the network, policy, and blockchain elements work together to empower someone to actually build an application.

Is that what you're thinking?

**Comment by michwill** [comment]
Yes, pretty much a recap of our discussions and wiki. Although, we don't have a public description of umbral algorithm also :-)

**Comment by michwill** [comment]
I guess, I'll draft something first, so don't worry
